content/account/organizations.textile

```diff
@@ -0,0 +1,178 @@
+
+h2(#primary). Primary account
+
+The primary account is an organization's main account and holds the following privileges:
+
+* The highest level of access to the organization.
+* Ownership of all all accounts within the organization.
+* The default account used for provisioning.
+
+h2(#accounts). Create accounts in an organization
+
+Create "accounts":https://ably.com/accounts/any in an organization:
+
+* Open the *Account* navigation dropdown.
+** Click *Organization Accounts*.
+** Click *New account*.
+** Add an account name and *Create account*.
+
+h3(#isolate). Isolate environments
+
+Use separate accounts within an organization to create isolated environments, such as production, staging, and development. Assign each environment a "package":/pricing#packages that meets its specific needs. For example, production may need high capacity with an Enterprise package, staging might use a Standard package, and development a Free package.
+
+h2(#manage). Manage user account access
+
+"Managing user":/account/users account access allows control of both organizational management permissions and access to specific accounts within your organization.
+
+Manage user account access:
+
+* Open the *Account* navigation dropdown.
+** Click *Organization Users* to view the list of users.
+** Select the *user* whose access you want to manage.
+** Specify the user's required *Permissions* for the account.
+** Assign the necessary *Roles* to grant the user access to specific *Accounts*.
+** Click *Back to Organization users* to save your changes.
+* Add the user to a new account.
+** Enter the account name.
+** Click *Create Account*.
+
+<aside data-type='note'>
+<p>Users provisioned through SCIM cannot modify their name or email address within Ably. All personal information updates must be made through your identity provider, and changes will sync to Ably upon the next SCIM update cycle.</p>
+</aside>
+
+h3(#permissions). User permissions
+
+A user's total permissions consist of:
+
+* Roles and permissions assigned directly to the user.
+* Roles and permissions granted to all groups the user is part of.
+
+h2(#scim). Provision users via SCIM
+
+<aside data-type='note'>
+<p>Google Workspace _alone_ does not natively support SCIM.</p>
+</aside>
+
+"Manage access":/account/users to multiple Ably accounts through a single identity provider. To enable this, configure both "SSO":/account/sso with your chosen identity provider and "SCIM":#SCIM. Once configured, Ably automatically provisions and deprovisions users with access to the Ably app in your identity provider, either individually or through assigned groups. New users are added to Ably's default provisioning account with the role of *Developer*.
+
+<aside data-type='important'>
+	<p>Ably will reject any SCIM-provisioned user whose email address hasn't been pre-approved and added to your organization by Ably support. Ensure all users you intend to provision are registered with Ably beforehand.</p>
+	</aside>
```

Comment:
I'm still questioning this. Is this the email address format that we're referring to? Or is it that each user should have created an account with Ably first? I can't believe support need to pre-approve every user which is how this reads.